src/styles/_markdown-theme.scss

```diff
@@ -17,5 +17,25 @@
     background: rgba(mat-color($foreground, secondary-text), .01);
     border: .5px solid rgba(mat-color($foreground, secondary-text), .03);
   }
+
+  .docs-markdown-h3,
+  .docs-markdown-h4 {
+    position: relative;
+    padding-left: 30px;
+    margin-left: -30px;
```

Comment:
Want the anchor to position 30px to the left of the header. Using absolutely position makes the anchor link not clickable because its outside the header hover.

Added comments for this.